Basic/2DSpaceShooter/ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2021.3.24f1
-m_EditorVersionWithRevision: 2021.3.24f1 (cf10dcf7010d)
+m_EditorVersion: 2022.2.13f1
```

Comment:
At this point 2023.3.0f1 is available. I'd suggest updating to it directly

Reply:
Absolutely. I wanted to run CI on 2022.2 to validate nothing had been introduced. Will update to 2022.3 LTS soon.